tests/user-test.js

```diff
@@ -1,7 +1,8 @@
 const chai = require('chai');
 const expect = chai.expect;
 
-const User = require('./src/User.js');
+const User = require('../src/User');
+
```

Comment:
I was a little hazy on the require syntax here, I remembered Khalid telling us a cleaner/shorter version for requiring, but couldn't remember the exact syntax.

Reply:
I also had to look at a prior project because it was hazy for me as well.